llvm-external-projects/iree-dialects/lib/Dialect/IREEPyDM/Transforms/ToIREE/LoweringPatterns.cpp

```diff
@@ -60,41 +64,34 @@ static Block *createSlowPathBlock(OpBuilder &builder) {
   return builder.createBlock(parentRegion, parentRegion->end());
 }
 
-static Type getVariantListType(Builder &builder) {
-  return builder.getType<iree_d::ListType>(
-      builder.getType<iree_d::VariantType>());
-}
-
 static Value getSuccessStatusValue(Location loc, OpBuilder &builder) {
-  return builder.create<arith_d::ConstantOp>(loc, builder.getI32IntegerAttr(0));
+  return builder.create<arith::ConstantOp>(loc, builder.getI32IntegerAttr(0));
 }
 
 static Value getFailureStatusValue(Location loc, OpBuilder &builder,
                                    ExceptionCode code) {
-  return builder.create<arith_d::ConstantOp>(
+  return builder.create<arith::ConstantOp>(
       loc, builder.getI32IntegerAttr(static_cast<int>(code)));
 }
 
 static Value createUndefObjectList(Location loc, OpBuilder &builder) {
-  return builder.create<iree_d::ListCreateOp>(loc, getVariantListType(builder),
-                                              /*capacity=*/nullptr);
+  return builder.create<iree::ListCreateOp>(loc, getVariantListType(builder),
+                                            /*capacity=*/nullptr);
 }
 
 void resetObjectList(Location loc, OpBuilder &builder, Value list, int typeCode,
                      Value data) {
   // Note: The list can record optional runtime state at positions > 1, so
   // to truly reset, we have to resize. Low level optimizations should be able
   // to elide this if it turns out to be unnecessary.
-  auto size = builder.create<arith_d::ConstantOp>(loc, builder.getIndexAttr(2));
-  builder.create<iree_d::ListResizeOp>(loc, list, size);
-  auto index0 =
-      builder.create<arith_d::ConstantOp>(loc, builder.getIndexAttr(0));
-  Value typeCodeValue = builder.create<arith_d::ConstantOp>(
+  auto size = builder.create<arith::ConstantOp>(loc, builder.getIndexAttr(2));
```

Comment:
these can be ConstantIndexOp, which just takes an int64_t so no need for attr stuff

Reply:
Thanks - knew that and not sure why I wrote it this way :/ Fixed.